static/js/jira-manual-app-creation.js

```diff
@@ -104,7 +104,7 @@ $(document).ready(function() {
 			if (isUpdatePage()) {
 				gitHubAppPutRequest(uuid, data);
 			} else {
-				gitHubAppPostRequest(data);
+				gitHubAppPostRequest(data, token);
```

Comment:
I understand the `data.jwt` is equal to `token`, but prefer this as more explicit as for the reason bellow:

IIRC, $.ajax({data}) will send the data as body, which means jwt is also included in body. Usually, token like stuff is better to be in the Auth header. 

So we might need another PR to cleanup the jwt from the body.